database_cleanup/models/purge_tables.py

```diff
@@ -6,6 +6,12 @@
 from ..identifier_adapter import IdentifierAdapter
 
 
+_TABLE_TYPE_SELECTION = [
+    ('BASE TABLE', 'Base Table'),
+    ('VIEW', 'View'),
```

Comment:
Please normalize DB selection value to lower and without spaces, and also be more specific in the label for avoiding misunderstandings with `ir.ui.view` records:
```suggestion
    ('base', 'SQL Table'),
    ('view', 'SQL View'),
```
I know you are using this value for using such `SQL code` in the query, but you can perfectly put there both values without using this list.